mappings/net/minecraft/advancement/SimpleAdvancement.mapping

```diff
@@ -1,5 +1,5 @@
 CLASS q net/minecraft/advancement/SimpleAdvancement
-	CLASS q$a Builder
+	CLASS q$a Task
```

Comment:
renamed this as there is `SimpleAdvancement#toTask` method mapped.

Reply:
I don't know anything about this class, so I have no inclination either way: does it make more sense to update the method to match the class, or to match the class with the method?

Reply:
The `toString` says this is "Task Advancement", and this thing, compared to the simple advancement object, is a pojo and can be more easily serialized.